Documentation/distributing_games.md

```diff
@@ -0,0 +1,124 @@
+
+```
+YourGame.app                    (this is your root folder)
+    - Contents
+        - Resources
+            - Content           (this is where all your content and XNB's should go)
+            - YourGame.icns     (this is your app icon, in ICNS format)
+        - MacOS                 (this is where your game belongs, except for content files)
+    - Info.plist                (the metadata of your app)
+```
+
+The Info.plist file is a standard macOS file containing metadata about your game. An example file would go like this:
+
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+    <key>CFBundleDevelopmentRegion</key>
+    <string>en</string>
+    <key>CFBundleExecutable</key>
+    <string>YourGame</string>
+    <key>CFBundleIconFile</key>
+    <string>YourGame</string>
+    <key>CFBundleIdentifier</key>
+    <string>com.my-domain.YourGame</string>
+    <key>CFBundleInfoDictionaryVersion</key>
+    <string>6.0</string>
+    <key>CFBundleName</key>
+    <string>YourGame</string>
+    <key>CFBundlePackageType</key>
+    <string>APPL</string>
+    <key>CFBundleShortVersionString</key>
+    <string>1.0</string>
+    <key>CFBundleSignature</key>
+    <string>FONV</string>
+    <key>CFBundleVersion</key>
+    <string>1</string>
+    <key>LSApplicationCategoryType</key>
+    <string>public.app-category.games</string>
+    <key>LSMinimumSystemVersion</key>
+    <string>10.13</string>
+    <key>NSHumanReadableCopyright</key>
+    <string>Copyright © 2020</string>
+    <key>NSPrincipalClass</key>
+    <string>NSApplication</string>
+</dict>
+</plist>
+```
+
+After completing these steps, your .app folder should appears as an executable application on macOS.
+
+For archiving, we recommend using the .tar.gz format for preserve the execution permissions.
+
+### Special notes about .Net Core parameters
+
+.Net Core proposes several parameters when publishing apps that may sound helpful, but have many issues when it comes to games (because they were never meant for games in the first place, but for small lightweight applications).
+
+**Ready2Run**
+
+You might be tempted to use Ready2Run assemblies for games. We recommended to not using it because it produces a lot a micro stutters throughout games.
+
+This is due to the fact that Ready2Run code is of low quality and makes the Just-In-Time compiler to trigger very regularly to promote the code to a higher quality. Whenever the JIT runs, it produces potentially very visible stutters that players don't like at all.
+
+Disabling Ready2Run solves this issue (at the cost of a slightly longer startup time, but typically very negligible).
+
+**TieredCompilation**
+
+Tiered compilation is a companion system to Ready2Run and works on the same principle to enhance startup time. We suggest disabling it to avoid any stutter while playing.
```

Comment:
```suggestion
[Tiered compilation](https://docs.microsoft.com/en-us/dotnet/core/whats-new/dotnet-core-3-0#tiered-compilation) is a companion system to ReadyToRun and works on the same principle to enhance startup time. We suggest disabling it to avoid any stutter while your game is running.
```